ssm/ssm2.py

```diff
@@ -551,7 +553,15 @@ def handle_connect(self):
         connect to each in turn until successful.
         """
         if self._protocol == Ssm2.AMS_MESSAGING:
-            log.debug('handle_connect called for AMS, doing nothing.')
+            log.info("Using AMS version %s", pkg_resources.get_distribution('argo_ams_library').version)
+
+            log.info("Establishing connection to %s", self._brokers[0])
+
+            if self._dest is not None:
+                log.info('Will send messages to: %s', self._dest)
+
+            if self._listen is not None:
+                log.info('Subscribing to: %s', self._listen)
```

Comment:
As per my comment on the broker, maybe something more like "Will subscribe to...."

Reply:
Thanks, changes made.